Dockerfile

```diff
@@ -178,11 +178,12 @@ COPY rootfs/rootfs $ROOTFS
 
 # Build the Hyper-V KVP Daemon
 RUN cd /linux-kernel && \
-    make headers_install INSTALL_HDR_PATH=/usr && \
+    make INSTALL_HDR_PATH=/tmp/kheaders headers_install && \
     cd /linux-kernel/tools/hv && \
-    sed -i 's/\(^CFLAGS = .*\)/\1 -m32/' Makefile && \
+    sed -i 's/\(^CFLAGS = .*\)/\1 -m32 -I\/tmp\/kheaders\/include/' Makefile && \
```

Comment:
Agreed! I didn't want to bring that change into this one, for example [line 95](https://github.com/boot2docker/boot2docker/blob/master/Dockerfile#L95) could use a different separator as well.

Should I include it or create a separate PR for that?